charts/nexus-iq/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: nexus-iq-server
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 143.0.0
+version: 143.0.1
```

Comment:
there's no need to bump this version in the PR. the job that does releases takes the version as a parameter and will update it at release-time.